webclient/webclient/src/main/java/io/helidon/webclient/NettyClientHandler.java

```diff
@@ -67,6 +68,7 @@ class NettyClientHandler extends SimpleChannelInboundHandler<HttpObject> {
      * Instance of the publisher used to handle response.
      */
     static final AttributeKey<BufferedEmittingPublisher> PUBLISHER = AttributeKey.valueOf("publisher");
+    static final AttributeKey<Flow.Publisher<DataChunk>> REQUEST_ENTITY = AttributeKey.valueOf("requestEntity");
```

Comment:
Please reuse the one from `WebClientRequestBuilderImpl` instead of creating the new one.